cypress/utils/utils.ts

```diff
@@ -53,3 +53,28 @@ export function checkSuccessAlert(fieldId: string, message: string): void {
 export function removeMember(memberName: string): void {
     cy.get("span").contains(memberName).siblings(commonView.removeButton).click();
 }
+
+export function exists(value: string) {
+    // Wait for DOM to render table and sibling elements
+    cy.wait(2000);
+    cy.get(commonView.appTable)
+        .next()
+        .then(($div) => {
+            if (!$div.hasClass("pf-c-empty-state")) {
+                cy.wait(2000);
+                cy.get("td").should("contain", value);
+            }
+        });
+}
+
+export function notExists(value: string) {
+    // Wait for DOM to render table and sibling elements
+    cy.wait(2000);
+    cy.get(commonView.appTable)
+        .next()
+        .then(($div) => {
+            if (!$div.hasClass("pf-c-empty-state")) {
+                cy.get("td").should("not.contain", value);
+            }
+        });
+}
```

Comment:
@sshveta @nitishSr 
As per my understanding, I think exists and notExists are related to object of that particular entity. 
Can I know the reason behind having these methods here instead of in model of that particular entity?

Reply:
@ganeshhubale : You are right , exists and not exists applies for an object but in cypress framework this is just a method that takes the name of an element and checks if it is there . 
All Exist methods will look same .So we can use one common method.

In earlier framework there were individual exist method because they their own navigation to element .

Reply:
As you pointed earlier, the assertions should be out of class. And going on same lines, this can belong to class, but it is assertion which is being repeated in all classes. The code re-usability is major concern if we go by previous approach. Because ultimately it is checking the existence or non existence of any entity in the table